waspc/cli/exe/Main.hs

```diff
@@ -150,6 +155,8 @@ printDbUsage =
         "  wasp db <command> [command-args]",
         "",
         title "COMMANDS",
+        cmd "  start       Starts development database.",
```

Comment:
Should we mention Docker here explicitly? May be nice to give them the "how" as well.

Reply:
Maybe we could -> I was thought thinking about it more as an implementation detail, you know they don't need to know how it is started. Although it will complain if they don't have docker :D. How would you write it? "Starts development database (via Docker)"?

Reply:
I added the check for "is docker available", which prints a nice error message if it is not! In that case, is it ok that I don't mention Docker here?